operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -434,7 +434,7 @@ def validateCluster(self, cluster):
       self.validateNonDynamicCluster(cluster)
     else:
       if self.env.isFMWInfraDomain() and not self.env.allowDynamicClusterInFMWInfraDomain():
-        self.addError("Dynamic clusters are not supported in FMW Infrastructure domains. Set ALLOW_DYNAMIC_CLUSTER_IN_FMW environment variable to true to bypass this validation.")
+        self.addError("Dynamic WebLogic clusters are not supported in FMW Infrastructure domains. Set ALLOW_DYNAMIC_CLUSTER_IN_FMW environment variable to true to bypass this validation.")
```

Comment:
The correct term here would be, "WebLogic dynamic clusters" ( as opposed to WebLogic configured clusters), not "Dynamic WebLogic clusters." (Please make this correction globally.)

Reply:
done